src/Microsoft.AspNetCore.WebHooks.Receivers/WebHookConstants.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.WebHooks.Routing;
+
+namespace Microsoft.AspNetCore.WebHooks
+{
+    /// <summary>
+    /// Well-known names and values used in WebHook receivers and handlers.
+    /// </summary>
+    public static class WebHookConstants
+    {
+        /// <summary>
+        /// Gets the minimum length of the <see cref="CodeQueryParameterName"/> query parameter value and secret key
+        /// for receivers using such a query parameter.
+        /// </summary>
+        /// <seealso cref="Metadata.IWebHookSecurityMetadata.VerifyCodeParameter"/>
+        public static int CodeParameterMinLength => 32;
+
+        /// <summary>
+        /// Gets the maximum length of the <see cref="CodeQueryParameterName"/> query parameter value and secret key
+        /// for receivers using such a query parameter.
+        /// </summary>
+        /// <seealso cref="Metadata.IWebHookSecurityMetadata.VerifyCodeParameter"/>
+        public static int CodeParameterMaxLength => 128;
+
+        /// <summary>
+        /// Gets the name of a query parameter containing the secret key used to verify some WebHook receivers'
+        /// requests.
+        /// </summary>
+        /// <seealso cref="Metadata.IWebHookSecurityMetadata.VerifyCodeParameter"/>
+        public static string CodeQueryParameterName => "code";
+
+        /// <summary>
+        /// <para>
+        /// Gets the key used to retrieve a configuration entry indicating the
+        /// <see cref="Filters.WebHookSecurityFilter.EnsureSecureConnection"/> should not ensure a secure (HTTPS)
+        /// connection with the sender. Configuration value, if any, should parse as a <see cref="bool"/>. If that
+        /// parsed configuration value is <see langword="true"/>, the HTTPS check is disabled. Otherwise i.e. if the
+        /// configuration value does not exist, cannot be parsed, or parses as <see langword="false"/>,
+        /// <see cref="Filters.WebHookSecurityFilter.EnsureSecureConnection"/> performs the check.
+        /// </para>
+        /// <para>
+        /// Key corresponds to an <see cref="Extensions.Configuration.IConfiguration"/> entry in the default
+        /// <see cref="IWebHookReceiverConfig"/> implementation
+        /// </para>
+        /// </summary>
+        /// <remarks>
+        /// Most, if not all, receiver configurations include
+        /// <see cref="Filters.WebHookSecurityFilter.EnsureSecureConnection"/> calls. For example,
+        /// <see cref="Filters.WebHookVerifyCodeFilter"/> calls the method if
+        /// <see cref="Metadata.IWebHookSecurityMetadata.VerifyCodeParameter"/> is <see langword="true"/>.
+        /// </remarks>
+        public static string DisableHttpsCheckConfigurationKey => "MS_WebHookDisableHttpsCheck";
```

Comment:
>public static string DisableHttpsCheckConfigurationKey => "MS_WebHookDisableHttpsCheck"; [](start = 8, length = 88)

What are these keys into, this isn't the way our IConfiguration system typically works